.husky/commit-msg

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+npx --no-install commitlint --edit "$1"
```

Comment:
Should be moved to `package.json` (`"lint-commit-msg": "npx --no-install commitlint --edit "$1""`).
Then this script becomes declarative (`yarn lint-commit-msg`) and all scripts are defined in a single location (`package.json`) and independent of the tools that use them